### PR TITLE
MM-16232 Add E2E to verify system config for Push Notification Contents

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Add E2E to verify system config for Push Notification Contents

Test steps:
1. Update to default config
2. Login as sysadmin and visit Notifications admin console page
- Verify that setting is visible and matches text content
- Verify that the help text is visible and matches text content
- Verify that the option/dropdown is visible and has default value
3. Select each option and save
- Verify that the config is correctly saved in the server

#### Ticket Link
Jira ticket: [MM-16232](https://mattermost.atlassian.net/browse/MM-16232)

Related PR - https://github.com/mattermost/mattermost-webapp/pull/4210

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -458,7 +458,7 @@
...
+  "admin.environment.notifications.pushContents.help": "**Generic description with only sender name** - Includes only the name of the person who sent the message in push notifications, with no information about channel name or message contents.\n **Generic description with sender and channel names** - Includes the name of the person who sent the message and the channel it was sent in, but not the message contents.\n **Full message content sent in the notification payload** - Includes the message contents in the push notification payload that is relayed through Apple's Push Notification Service (APNS) or Google's Firebase Cloud Messaging (FCM). It is **highly recommended** this option only be used with an \"https\" protocol to encrypt the connection and protect confidential information sent in messages.\n**Full message content fetched from the server on receipt** - The notification payload relayed through APNS or FCM contains no message content, instead it contains a unique message ID used to fetch message content from the server when a push notification is received by a device. If the server cannot be reached, a generic notification will be displayed.",
```
Comment (saturninoabril) [inline i18n/en.json]: Removes unnecessary space after `-` (just leave it as single space)
from:
```
payload** -  Includes the message
```
to:
```
payload** - Includes the message
```

File: e2e/cypress/integration/system_console/ui_and_api/notifications_spec.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+                'Full message content fetched from the server on receipt',
+                ' - The notification payload relayed through APNS or FCM contains no message content, instead it contains a unique message ID used to fetch message content from the server when a push notification is received by a device. If the server cannot be reached, a generic notification will be displayed.',
+            ];
+            cy.wrap(el).should('have.text', contents.join(''));
+
+            cy.wrap(el).find('strong').eq(0).should('have.text', contents[0]);
+            cy.wrap(el).find('strong').eq(1).should('have.text', contents[2]);
+            cy.wrap(el).find('strong').eq(2).should('have.text', contents[4]);
+            cy.wrap(el).find('strong').eq(3).should('have.text', contents[6]);
+            cy.wrap(el).find('strong').eq(4).should('have.text', contents[8]);
```
Comment (srkgupta) [inline e2e/cypress/integration/system_console/ui_and_api/notifications_spec.js]: As advised by Bob, we do not need to include cy.wrap again, inside the within block. This code could be simplified like this:

```
            cy.get('strong').eq(0).should('have.text', contents[0]);
            cy.get('strong').eq(1).should('have.text', contents[2]);
            cy.get('strong').eq(2).should('have.text', contents[4]);
            cy.get('strong').eq(3).should('have.text', contents[6]);
            cy.get('strong').eq(4).should('have.text', contents[8]);
```

Comment (saturninoabril) [reply e2e/cypress/integration/system_console/ui_and_api/notifications_spec.js]: Done, PTAL

**Comment by enahum** [comment]
After a follow up [PR](https://github.com/mattermost/mattermost-webapp/pull/4293) this one is going to fail.

**Comment by saturninoabril** [comment]
Thanks for the heads up.  I'll take care of it.

**Pull request merged**